### ISSUE TITLE
Keycloak Admin.v2 - Error when copy in new page realm-management --> Authorization --> Policies

### ISSUE THREAD

**Issue by mpalavrov** [keycloak/keycloak-ui#3603]
### Describe the bug

When you open page **realm-management --> Authorization --> Policies** and then you copy and paste in new tab/window, there is an error displayed

### Version

19.0.2

### Environment

Firefox, Chrome, etc.

### Expected behavior

The page to be opened

### Actual behavior

Error `(intermediate value)().clientInstallations is undefined` is displayed

### How to Reproduce?

    1. Go to Clients --> realm-management --> Authorization --> Policies
    2. Copy the page URL
    3. Paste the page URL in new tab/window

**Comment by ssilvert** [comment]
@mpalavrov I can't reproduce this one.  Can you try it on the nightly build and let us know if it works?

https://github.com/keycloak/keycloak/releases/tag/nightly

**Comment by mpalavrov** [comment]
Hi @ssilvert,
Yes, it seems that it is not reproducible with the https://github.com/keycloak/keycloak/releases/tag/nightly. Something I noticed using it is that the `Clients --> realm-management --> Authorization` menu was not shown
![image](https://user-images.githubusercontent.com/33352511/196867677-45a4199a-3314-4dd9-ba5a-48cb08374c60.png)
but was shown only after I added "/authorization" in the URL, and then it became visible even after opening in the UI. If this was something only I observed, it is not an issue, and also the initially reported by my seems to be already fixed in this version.
Thank you for checking.
Best,
Milen

**Comment by ssilvert** [comment]
For most clients, you will see the `Authorization` tab if authorization is enabled on the `Settings` tab.

For `realm-management` it should be enabled by default.  I can see it.  Since this would be a different issue, please open a new issue if you continue to have problems.  I'm going to close this one.
